.github/actions/templates/deployModule/action.yml

```diff
@@ -51,7 +51,10 @@ runs:
         # Load used funtions
         . ".github/actions/sharedScripts/Convert-TokensInFiles.ps1"
         $TokensReplaceWithObject = @(
-            @{ Replace = "<<subscriptionId>>"; With = '${{ inputs.subscriptionId }}' }
+            @{ Replace = "<<tenantId>>"; With = '${{ env.ARM_TENANT_ID }}' }
+            @{ Replace = "<<subscriptionId>>"; With = '${{ env.ARM_SUBSCRIPTION_ID }}' }
+            @{ Replace = "<<managementGroupId>>"; With = '${{ env.ARM_MGMTGROUP_ID }}' }
+            @{ Replace = "<<principalId>>"; With = '${{ env.ARM_PRINCIPAL_ID }}' }
```

Comment:
I believe we can should plan ahead that one might want to test with multiple different principals. I'd hence suggest to rename this to e.g. `@{ Replace = "<<principalId1>>"; With = '${{ env.ARM_PRINCIPAL_ID_1 }}' }`

Reply:
If updated, the change has to happen across the other files too, of course

Reply:
Yes. That's fine we have to update this on both Deploy and Validate actions.

Reply:
Looks good, thanks